scripts/screeps_rl_training_runner.py

```
@@ -1897,6 +2002,488 @@ def mean_policy_return_tuple(return_tuples: Sequence[Sequence[Any]]) -> list[flo
+            raw,
+            (
+                "gradient_reward_weights",
+                "gradientRewardWeights",
+                "scalar_reward_weights",
+                "scalarRewardWeights",
+            ),
+        )
+        if component_weights is not None:
+            source_weights.update(policy_update_component_weights_by_tier(component_weights))
+
+    normalization_factor = max(max(abs(value) for value in source_weights.values()), 1.0)
```

Ruff (0.15.13)

[warning] Nested `max` calls can be flattened

Flatten nested `max` calls

(PLW3301)